models/orange_pasta/configs/config_hyperparameters.py

```diff
@@ -1,7 +1,13 @@
 def get_hp_config(): 
     hp_config = {
-        "learning_rate": 0.05,
-        "n_estimators": 100,
-        "n_jobs": 12   
+        "name": "orange_pasta",
+        "algorithm": "LGBMRegressor",
+        "depvar": "ged_sb_dep",
```

Comment:
Again, these do not seem operational and might below in he config_meta

Reply:
I actually need these parameters to generalize codes, so changing them will affect the process. For example, the name is used to initialize the project name; the algorithm is used to initialize a model. Otherwise I have to hardcode them.

Reply:
Yeah, no hard coding of course. So I wrote a whole thing about this in slack at some point, but since I didn't it down somewhere more official I get why it might have been missed. But I have now put it here:

https://github.com/prio-data/views_pipeline/blob/main/documentation/ADRs/002_seperation_of_configs.md

A new documentation sub-folder mostly for internal use. This readme should explain why:

https://github.com/prio-data/views_pipeline/blob/main/documentation/ADRs/README.md

Reply:
Also, I can see that three of these are already in the meta config (where I think they below):

- "name": "orange_pasta", (already in meta)
  "algorithm": "LGBMRegressor", (already in meta)
  "depvar": "ged_sb_dep", (already in meta)
  
So I really just ask that you take them from there and not the other config (according to the ADR). 

I'm not sure about "steps": [*range(1, 36 + 1, 1)]. I have a similar parameter for Hydranet in the hyperparameters and sweep config... I'm not sure how I feel about that but I think we can both leave it for now.